CMakeLists.txt

```diff
@@ -2,22 +2,48 @@
 cmake_minimum_required(VERSION 3.4)
 project(libuv LANGUAGES C)
 
+cmake_policy(SET CMP0057 NEW)
```

Comment:
Can you perhaps add a one-liner above each, explaining what they do? a link to the CMake docs is also a good option.

Reply:
Sure thing! I'll push up some changes soon(ish)